backstop-non-responsive-ce.json

```diff
@@ -34,6 +34,26 @@
       "misMatchThreshold": 0,
       "requireSameDimensions": true
     },
+    {
+      "label": "buttons",
+      "cookiePath": "",
+      "url": "http://localhost:8000/tests/custom-elements/buttons.html",
+      "referenceUrl": "",
+      "readyEvent": "",
+      "readySelector": "",
+      "delay": 0,
+      "hideSelectors": [],
+      "removeSelectors": [],
+      "hoverSelector": "",
+      "clickSelector": "",
+      "postInteractionWait": "",
+      "selectors": [
+        ".test-outline"
```

Comment:
@srinath-koneru This tests should mirror the ones in backstop-non-responsive.json (line: 63)